reana/reana_dev/run.py

```diff
@@ -476,12 +477,37 @@ def run_example(
 
     display_message(
         "Running the following matrix:\n"
-        "Demos: {}\n"
-        "Workflow engines: {}\n"
-        "Compute backends: {}".format(
-            ",".join(components), ",".join(workflow_engines), ",".join(compute_backends)
-        )
+        f"Demos: {','.join(components)}\n"
+        f"Workflow engines: {','.join(workflow_engines)}\n"
+        f"Compute backends: {','.join(compute_backends)}"
     )
+
+    def _verify_log_output(component: str, workflow_name: str) -> bool:
+        for log_message in get_expected_log_messages_for_example(component):
+            cmd = f"reana-client logs -w {workflow_name} | {{ grep -c '{log_message}' || true; }}"
+            count = int(run_command(cmd, component, return_output=True))
```

Comment:
Cosmetics: Currently people see:

```console
[2022-04-26T10:01:58] reana-demo-helloworld: reana-client status -w helloworld-snakemake-kubernetes
[2022-04-26T10:01:58] reana-demo-helloworld: reana-client status -w helloworld-yadage-kubernetes
[2022-04-26T10:01:59] reana-demo-helloworld: reana-client status -w helloworld-serial-kubernetes
[2022-04-26T10:02:00] reana-demo-helloworld: reana-client logs -w helloworld-serial-kubernetes | { grep -c 'Parameters: inputfile=' || true; }
```

i.e. the commands don't have any visible output.

What about detecting the output and then explicitly printing it, so that people would see _something_ and have in this way some visual feedback? In this way the output of this command, which may live in some text file, will be more informative. For example:

```console
[2022-04-26T10:01:59] reana-demo-helloworld: reana-client status -w helloworld-serial-kubernetes
NAME                           RUN_NUMBER   CREATED               STARTED               ENDED                 STATUS     PROGRESS
helloworld-serial-kubernetes   12           2022-04-26T08:01:14   2022-04-26T08:01:32   2022-04-26T08:01:42   finished   1/1
[2022-04-26T10:02:00] reana-demo-helloworld: reana-client logs -w helloworld-serial-kubernetes | { grep -c 'Parameters: inputfile=' || true; }
1
```

IOW, you can capture the output for analysis, obviously, but then please emit it too, for users's convenience.

Reply:
Actually, in my case, I can see the output.

![Screenshot 2022-04-26 at 12 04 39](https://user-images.githubusercontent.com/15694232/165276323-03828d73-bdb5-4748-b80e-4a862f71cf5a.png)

Reply:
Oh. Now I see. I will make it print the results.

Reply:
I added print at the end. But it will only print a number of lines.